PlayRho/Collision/Manifold.hpp

```diff
@@ -577,9 +574,10 @@ Manifold GetManifold(bool flipped,
                      const Manifold::Conf conf);
 
 /// @brief Computes manifolds for face-to-point collision.
-Manifold GetManifold(bool flipped, Length totalRadius,
-                     const DistanceProxy& shape, const Transformation& sxf,
-                     Length2 point, const Transformation& xfm);
+/// @pre The given distance proxy <code>GetVertexCount()</code> must be one or greater.
+/// @warning Behavior is undefined if the given distance proxy <code>GetVertexCount()</code> is less than one.
+Manifold GetManifold(bool flipped, Length totalRadius, const DistanceProxy& shape,
+                     const Transformation& sxf, Length2 point, const Transformation& xfm);
```

Comment:
This seems to have gotten flagged here and not from my changes to the master branch because I changed the format of this declaration so it more closely matched what it has been in the master branch and this reviewer seems to focus on the changes in the pull request.

In general, I want to move to the trailing return type style but I'm more in favor of that for version 2+ releases. I'm going to adjust the formatting back to the way it was for this function so this return style hopefully won't get suggested again.

Also, I think I prefer the idea of the clang-tidy-review action only making suggestions and not failing as checks go just due to issuing any comments.